src/js/Controllers/TTSController.js

```diff
@@ -207,10 +206,15 @@ class TTSController {
         }
     }
 
-    onSpeakTimeout(msgID) {
-        delete this.timers[msgID]
-
-        this.listener.send(RpcFactory.TTSSpeakResponse({ id: msgID, method: 'TTS.Speak' }))
+    isSpeakFinished(msgID) {
+        var file = this.filePlaylist.shift();
```

Comment:
you  mutated `this.filePlaylist` here, and then use it with removed first element. Is it expected behaviour?